README.md

```diff
@@ -306,6 +306,10 @@ If you are using a different method for authentication, see the examples below:
 
   * `{login_name_for_IdP}` is your login name for your IdP.
 
+  You can also override the default timeout after which external browser authentication is marked as failed.
+  This prevents the situation in which client decides to quit a browser tab.
```

Comment:
How about `The timeout prevents the infinite hang when the user does not provide the login details, e.g. when closing the browser tab`?

Reply:
Wonderful!